packages/fxa-graphql-api/src/auth/unverified-session-guard.spec.ts

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { ExecutionContext, UnauthorizedException } from '@nestjs/common';
+
+const mockSession = {
+  SessionToken: {
+    findByTokenId: jest.fn(),
+  },
+};
+jest.mock('fxa-shared/db/models/auth/session-token', () => mockSession);
+
+// eslint-disable-next-line import/first
+import { UnverifiedSessionGuard } from './unverified-session-guard';
```

Comment:
Re: `eslint-disable`,

Any reason to `import` this here vs above the `mockSession` on L6-11?